pkg/gui/controllers/helpers/window_arrangement_helper.go

```diff
@@ -237,14 +238,13 @@ func mainSectionChildren(args WindowArrangementArgs) []*boxlayout.Box {
 }
 
 func getMidSectionWeights(args WindowArrangementArgs) (int, int) {
-	// we originally specified this as a ratio i.e. .20 would correspond to a weight of 1 against 4
 	sidePanelWidthRatio := args.UserConfig.Gui.SidePanelWidth
-	// we could make this better by creating ratios like 2:3 rather than always 1:something
-	mainSectionWeight := int(1/sidePanelWidthRatio) - 1
-	sideSectionWeight := 1
+	// Using 24 so that the default of 0.3333 will remain consistent with previous behavior
```

Comment:
Is the idea that we want this number to have many factors? If so, how about we add 5 as a factor to it to make 120?

Also we should have this value assigned to a variable rather than duplicate it